src/index.ts

```diff
@@ -127,9 +128,14 @@ class PgAnonymizer extends Command {
           .split("\t")
           .map((v, k) => {
             if (indices.includes(k)) {
-              const replacement = list.find(
+              let replacement = list.find(
                 (l) => l.col === cols[k]
               )?.replacement;
+              if (!replacement) {
+                replacement = list.find(
+                  (l) => l.col === table + '/' + cols[k]
+                )?.replacement;
+              }
```

Comment:
I can't understand my code anymore 😅 so I guess your implementation is OK then! I'm not sure I understand why there is a `if (!replacement)`, still it's the end of the year and I must be tired. Let's go!